### ISSUE TITLE
Prebuild / postbuild execute command or task

### ISSUE THREAD

**Issue by Invisibleman1002** [microsoft/vscode-arduino#1518]
In a tasks.json file you can have something like this:
```
{
      "label": "Stop Serial",
      "type": "process",
      "command": "${command:serialdevices.stop}"
}
```
Is there a way to have the build commands run a specific task or at least run a command from another extension?

This would be ideal, otherwise maybe be able to execute the above task,
```
"prebuild": "${command:serialdevices.stop}"
```

**Comment by StefanKellerAC** [comment]
Yes, at least an acoustic signal when build/upload is ready would be fine.
Especially with different signals, when Arduino exits with or without error.
